### ISSUE TITLE
Do the widgets render well in several parts of the layout?

### ISSUE THREAD

**Issue by IagoLast** [CartoDB/airship#456]
Create smoke.html files to have a quick summary about how the widgets are rendered in:

- sidebar
- sidebar-xl
- footer
- panel

**Comment by ivanmalagon** [comment]
Of course, if something weird is found it has to be fixed. First the research, then the fixes.

**Comment by ivanmalagon** [comment]
This has been closed but I miss a report on this. How can we run an acceptance on the issue if there's not clue of the `smoke.html` pointed in the description?

There's only a PR about stacked bar widget. Do we have to suppose that category and histogram rendered in those places with no problem?
